command/agent/config.go

```diff
@@ -77,6 +77,17 @@ type DNSConfig struct {
 	// returned by default for UDP.
 	EnableTruncate bool `mapstructure:"enable_truncate"`
 
+	// UDPAnswerLimit is used to limit the maximum number of DNS Resource
+	// Records returned in the ANSWER section of a DNS response. This is
+	// not normally useful and will be limited based on the querying
+	// protocol, however systems that implemented §6 Rule 9 in RFC3484
+	// may want to set this to 1 in order to achieve round-robin DNS.
+	// RFC3484 sorts answers in a deterministic order, which defeats the
+	// purpose of round-robin DNS.  This RFC has been obsoleted by
```

Comment:
Perhaps it should specify that it defeats round robin in the client.  So that it is clear that the client has the round robin issue.  If we are talking about consul then 'randomized DNS' is correct, if the client then 'round robin' is the distressed feature.